frappe/core/doctype/user/user.py

```diff
@@ -574,10 +574,9 @@ def after_rename(self, old_name, new_name, merge=False):
 			has_fields = [d.get("name") for d in desc if d.get("name") in ["owner", "modified_by"]]
 			for field in has_fields:
 				frappe.db.sql(
-					"""UPDATE `%s`
-					SET `%s` = %s
-					WHERE `%s` = %s"""
-					% (tab, field, "%s", field, "%s"),
+					"""UPDATE `{}`
+					SET `{}` = {}
+					WHERE `{}` = {}""".format(tab, field, "%s", field, "%s"),
```

Comment:
rewrite this with `frappe.db.set_value(..., update_modified=False)`

Reply:
Will do this in a separate PR since not directly related to ruff itself.